language-support/ts/codegen/tests/ts/generated/package.json

```diff
@@ -5,8 +5,9 @@
   "description": "Generated by daml2ts",
   "license": "Apache-2.0",
   "dependencies": {
+    "@daml/ledger": "0.0.0-SDKVERSION",
```

Comment:
`yarn add` does this autoformatting of `package.json` files. I suggest we all use `yarn add` to add new dependencies to avoid diff noise. It also looks up the newest version of a package for us.

Reply:
:+1: